lib/Conversion/ATenToXTenPass.cpp

```diff
@@ -145,6 +144,53 @@ bool isLongestBranch(OpResult left, OpResult right) {
   return true;
 }
 
+// Tablegen adapter to check if attributes allow a conversion from `ReduceMean`
+// to `GlobalAveragePool`.
+bool isReduceMeanGlobalAveragePool2D(Value dims, Value keepdims) {
+
+  // Get defining operands
+  Torch::PrimListConstructOp dimsOp =
+      llvm::dyn_cast<Torch::PrimListConstructOp>(dims.getDefiningOp());
+  Torch::ConstantBoolOp keepdimsOp =
+      llvm::dyn_cast<Torch::ConstantBoolOp>(keepdims.getDefiningOp());
+
+  if (!dimsOp || !keepdimsOp)
+    return false;
+
+  SmallVector<APInt> axes;
+  for (Value operand : dimsOp.getOperands()) {
+    if (auto axesValue =
+            llvm::dyn_cast<Torch::ConstantIntOp>(operand.getDefiningOp()))
+      axes.push_back(axesValue.value());
+  }
+
+  // Check that arguments match expectations for global average pool
+  if (keepdimsOp.value() == 1 && axes.size() == 2 && axes[0] == 2 &&
+      axes[1] == 3)
```

Comment:
Would:
```suggestion
  if (keepdimsOp.value() == 1 && axes == {2, 3})
```
work?

Reply:
You cannot use an initializer list on the right hand side of operator '==' ~